src/graphql/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  extends: ['../../.eslintrc.js'],
+  overrides: [
+    {
+      files: ['*.graphql'],
+      parser: '@graphql-eslint/eslint-plugin',
+      plugins: ['@graphql-eslint'],
+      rules: {
```

Comment:
Yeah, I initially tried to do this, however, I had trouble getting ESLint overrides to work well with multiple remote schemas. Thought this would be an okay start...

Reply:
Oh! Makes sense. If it's something we should do eventually let's just make a ticket for it 👍